### PR TITLE
disable signed-off-by header in Tycho setup

### PR DESCRIPTION
**Pull request by Bananeweizen**

### CODE REVIEW

File: setup/Tycho.setup [hunk 1]
```diff
@@ -32,8 +32,8 @@
       <setupTask
           xsi:type="setup:PreferenceTask"
           key="/instance/org.eclipse.egit.ui/commit_dialog_signed_off_by"
-          value="true">
-        <description>Have egit add the signed-off-by footer to the commit message by default</description>
+          value="false">
+        <description>Signed-off-by not used anymore with github</description>
```
Comment (mickaelistria) [inline setup/Tycho.setup:36]: It's not with GitHub, it's the Eclipse Development Process that was simplified, so it applies to whichever repository and codebase.

Comment (Bananeweizen) [reply setup/Tycho.setup]: Thanks for clarifying. I wasn't aware of that and thought it is based on whether gerrit or github is used.

**Comment by mickaelistria** [comment]
There is https://bugs.eclipse.org/bugs/show_bug.cgi?id=558653#c30 on this topic (I'm not sure it was widely announced or not).

**Comment by mickaelistria** [comment]
Thanks fo rthe change!

**Pull request merged**